src/main/steps/c100-rebuild/applicant/relationship-to-child/postController.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Can we reuse postController of respondent's relationship as sonar will throw duplicates.